framework/src/abi_handler/abi_handler.ts

```diff
@@ -541,7 +541,8 @@ export class ABIHandler implements ABI {
 			});
 			this._logger.info({ method: req.method }, 'Called ABI query successfully');
 			return {
-				data: Buffer.from(JSON.stringify(resp), 'utf-8'),
+				// default resp to '' for endpoints with no response
+				data: Buffer.from(JSON.stringify(resp ?? ''), 'utf-8'),
```

Comment:
```suggestion
				data: Buffer.from(JSON.stringify(resp ?? '{}'), 'utf-8'),
```
I think empty is not valid JSON, which will fail in parse

Reply:
It worked fine when I invoked the endpoint locally.

But I've force pushed a commit which changes default response to `'{}'`